BrainstormSessions/Infrastructure/EFStormSessionRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BrainstormSessions.Core.Interfaces;
+using BrainstormSessions.Core.Model;
+using Microsoft.EntityFrameworkCore;
+using Serilog;
+
+namespace BrainstormSessions.Infrastructure
+{
+    public class EFStormSessionRepository : IBrainstormSessionRepository
+    {
+        private readonly AppDbContext _dbContext;
+
+        public EFStormSessionRepository(AppDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public Task<BrainstormSession> GetByIdAsync(int id)
+        {
+            Log.Logger.Information($"Requested to get session with id: {id}.");
```

Comment:
In the repository classes logs should have debug level, because they will be written in huge amount